### PR TITLE
[ISSUE #3801] support clickhouse-logging-pugin

### PR DESCRIPTION
**Pull request by 847850277**
Dear community:
i have finished support clickhouse-logging-pugin and unit test.
pls review code.

Make sure that:

- [ ] You have read the [contribution guidelines](https://shenyu.apache.org/community/contributor-guide).
- [ ] You submit test cases (unit or integration tests) that back your changes.
- [ ] Your local test passed `./mvnw clean install -Dmaven.javadoc.skip=true`.

### CODE REVIEW

**Comment by yu199195** [comment]
pls resolve conflicts

**Comment by 847850277** [comment]
> resolve

it be resolved.

**Comment by 847850277** [comment]
> pls resolve conflicts


it be resolved.

File: db/upgrade/2.5.0-upgrade-2.5.1-pg.sql [hunk 1]
```diff
@@ -26,9 +30,7 @@ INSERT INTO "public"."plugin_handle" VALUES ('1529403902783524974', '35', 'sampl
 INSERT INTO "public"."plugin_handle" VALUES ('1529403902783524975', '35', 'maxResponseBody', 'maxResponseBody', 1, 3, 5, '{"required":"0","defaultValue":524288}', '2022-06-30 21:00:00', '2022-06-30 21:00:00');
 INSERT INTO "public"."plugin_handle" VALUES ('1529403902783524976', '35', 'maxRequestBody', 'maxRequestBody', 1, 3, 6, '{"required":"0","defaultValue":524288}', '2022-06-30 21:00:00', '2022-06-30 21:00:00');
 INSERT INTO "public"."plugin_handle" VALUES ('1529403902783524977', '35', 'compressAlg', 'compressAlg', 3, 3, 7, '{"required":"0","defaultValue":"none"}', '2022-06-30 21:00:00', '2022-06-30 21:00:00');
-
--- insert loggingTencentCls plugin
-INSERT INTO "public"."plugin" VALUES ('36', 'loggingTencentCls', '{"endpoint\": \"ap-guangzhou.cls.tencentcs.com\, \"topic\": \"shenyu-topic"}', 'Logging', 176, 0, '2022-06-30 21:00:00', '2022-06-30 21:00:00');
```
Comment (yu199195) [inline db/upgrade/2.5.0-upgrade-2.5.1-pg.sql:31]: 1

**Review by yu199195 — APPROVED** [review]

**Pull request merged**